options.js

```diff
@@ -5,7 +5,11 @@
     urls = result["scroll-mark"];
     for (var url in urls) {
       var div = document.createElement("div");
-      div.innerHTML = "<a href=" + url + ">" + url + "</a>";
+      let percentage = Math.round((urls[url].offset / urls[url].total) * 100);
+      if (percentage)
+        div.innerHTML =
+          "<a href=" + url + ">" + url + "</a> (" + percentage + "%)";
```

Comment:
```suggestion
          "<a href=" + url + ">" + title + "</a> (" + percentage + "%)";
```
Then we would be able to do something like this, which would look better I believe.

Reply:
Also, do handle the case where the title is not available, show the URL in that case.